rest-api-spec/src/main/resources/rest-api-spec/api/ml.put_trained_model.json

```
@@ -26,6 +26,14 @@
         }
       ]
     },
+    "params":{
+      "defer_definition_decompression": {
+        "required": false,
+        "type": "boolean",
+        "description": "Should the action skip decompressing the definition to validate it and set default values",
```

Since ideally the API docs will ultimately be generated from specs, I think this should match what's in the other asciidoc file. e.g. 
```suggestion
        "description": "If set to `true` and a `compressed_definition` is provided, the request defers definition decompression and skips relevant validations.",
```